bodhi/client/__init__.py

```diff
@@ -244,13 +246,24 @@ def _save_override(url, user, password, staging, edit=False, **kwargs):
         kwargs (dict): Other keyword arguments passed to us by click.
     """
     client = bindings.BodhiClient(base_url=url, username=user, password=password, staging=staging)
+
     resp = client.save_override(nvr=kwargs['nvr'],
                                 duration=kwargs['duration'],
                                 notes=kwargs['notes'],
                                 edit=edit,
                                 expired=kwargs.get('expire', False))
+    resp['wait'] = kwargs['wait']
```

Comment:
Since ```wait``` is not an attribute of an update, this is confusing. Why not just access wait directly below?